src/main/java/gov/cms/mat/cql_elm_translation/service/DataCriteriaService.java

```diff
@@ -135,10 +139,34 @@ private Set<String> getUsedDefinitionsFromMeasure(Measure measure) {
                           usedDefinitions.add(population.getDefinition());
                         }
                       });
+              if (group.getMeasureObservations() != null
```

Comment:
do we have access to CollectionUtils in this project? If so we can replace collection null/empty checks with just CollectionUtils.isEmpty or CollectionUtils.isNotEmpty

Reply:
switched it over